### PR TITLE
Btat 3442

### PR DESCRIPTION
**Pull request by davidmthornton**

### CODE REVIEW

File: app/views/hello_world.scala.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@*
+ @*
```
Comment (lewishayqa) [inline app/views/hello_world.scala.html]: Remove the indent here :)

File: app/controllers/LanguageController.scala [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import javax.inject.Inject
+import config.AppConfig
+import play.api.i18n.{Lang, MessagesApi}
+import play.api.mvc.{Action, AnyContent, Call}
+import uk.gov.hmrc.play.language.LanguageUtils
+
+class LanguageController @Inject()(val appConfig: AppConfig,
+                                   override val messagesApi: MessagesApi) extends BaseController {
+
+  def langToCall: String => Call = appConfig.routeToSwitchLanguage
```
Comment (lewishayqa) [inline app/controllers/LanguageController.scala]: The type of this def does not match the type of the appConfig def it's pointing to.

File: app/controllers/BaseController.scala [hunk 3]
```diff
@@ -0,0 +1,36 @@
+/*
```
[the remark this answers is not in the document]
Comment (davidmthornton) [reply app/controllers/BaseController.scala]: Removed

File: build.sbt [hunk 4]
```diff
@@ -1,27 +1,113 @@
-import TestPhases.oneForkedJvmPerTest
-import uk.gov.hmrc.DefaultBuildSettings.{addTestReportOption}
-import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
+import sbt._
```
[the remark this answers is not in the document]
Comment (davidmthornton) [reply build.sbt]: Removed

File: app/views/hello_world.scala.html (hunk 1 above)
Comment (davidmthornton) [reply app/views/hello_world.scala.html]: done

File: app/controllers/LanguageController.scala (hunk 2 above)
Comment (davidmthornton) [reply app/controllers/LanguageController.scala]: Fixed

File: app/controllers/LanguageController.scala [hunk 5]
```diff
@@ -0,0 +1,40 @@
+/*
```
[the remark this answers is not in the document]
Comment (davidmthornton) [reply app/controllers/LanguageController.scala]: done

**Comment by davidmthornton** [comment]
Ready for another review

**Pull request merged**